en-GB/lang/numere.nlng

```diff
@@ -364,6 +366,7 @@ PARSERFUNCS_LISTFUNC_METHOD_COVAROF_[DATA]=					covarof(nCol1,nCol2,nRows1={1:nr
 PARSERFUNCS_LISTFUNC_METHOD_RANKOF_[DATA]=					rankof(nCol,cType="c",nRows={1:nrows})                     {VAL}     - Calculates the ranks of the values in the column nCol for the rows nRows according the type cType. The rows can be specified and correspond to the full dimension by default. The type cType determines how ranks are assigned%%ITEMIZE%%cType = "c": competetively, i.e. according to the 1-2-2-4 scheme%%ITEMIZE%%cType = "f": fractionally, i.e. according to the 1-2.5-2.5-4 scheme%%ITEMIZE%%cType = "d": consecutive-dense, i.e. according to the 1-2-2-3 scheme%%ITEMIZE_END%%
 PARSERFUNCS_LISTFUNC_METHOD_DESCRIBE_[DATA]=				describe(sDesc)                                             STR      - Updates the description of the current table with the contents of sDesc. If this succeeds, sDesc is returned
 PARSERFUNCS_LISTFUNC_METHOD_DESCRIPTION_[DATA]=				description                                                 STR      - Returns the description of the current table as a string
+PARSERFUNCS_LISTFUNC_METHOD_FIRST_[STRING]=					endsw(sStr)                                                 STR      - Gives the value True back, when the second Substring is found at the end of the first string. Otherwise, False.

```

Comment:
```suggestion
PARSERFUNCS_LISTFUNC_METHOD_FIRST_[STRING]=					endsw(sEnd)                                                 LOG      - Returns true, if the string ends with sEnd. Otherwise, False.

```